### DISCUSSION TITLE
Is AutoFixture compatible with .NET 5.0?

### DISCUSSION THREAD

**Discussion by EmilThorEmilsson** [AutoFixture/AutoFixture#1255] [Q&A] [answered]
I see all the dependencies seem to be for .NET Framework older versions.
What about .NET 5.0 or .NET Core?

**Comment by aivascu** [comment, marked as answer]
@EmilThorEmilsson at this moment [AutoFixture ](https://www.nuget.org/packages/AutoFixture/) supports **.NET Standard 1.6** and **.NET Standard 2.0** which cover both **.NET Core** and **.NET 5** runtimes, as well as [any future versions](https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/#for-the-impatient-tldr) of .NET. 
To find out more about what .NET Standard is, you can visit this [page](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) on Microsoft's documentation website.
AutoFixture will add support for the newer C# features like records and init-only properties, in upcoming major versions, later this year.
